homework-submission/react-hyf/react-hyf-client/src/components/Homeworkweek3/index.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react';
+import List from './List';
+
+class HomeworkWeek3 extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      error: null,
+      isLoaded: false,
+      items: [],
+    };
+  }
+
+  componentDidMount() {
+    fetch('https://uinames.com/api/?amount=10')
+      .then(res => res.json())
+      .then(items =>
+        this.setState(
+          {
+            isLoaded: true,
+            items,
+          },
+          () => console.log(items),
+        ),
+      );
+  }
+
+  render() {
+    if (!this.state.isLoaded) {
+      return <div>Loading...</div>;
+    } else {
+      return (
+        <div>
+          <ul>
+            {this.state.items.map((item, i) => (
+              <div>
```

Comment:
There is no need for these `<div>`'s, to only clutter your HTML. It is also giving you an error (`Each child in a list should have a unique "key" prop.`) because the key is on the list and the `<div>` is the first element and the first element should have a key.

Reply:
@paulbremer Thanks for your feedback but the server file still not working

Reply:
Did you run `npm run build` in the client to build the index.html and then run the server?

Reply:
ja, It works a lot of thanks to you